src/components/blog/blog-content.tsx

```
@@ -0,0 +1,54 @@
+        .replace(/<\/li><\/p>/g, "</li>")
+    )
+  }
+
+  useEffect(() => {
+    setHtmlContent(markdownToHtml(content))
+  }, [content])
+
+  return (
+    <div
+      className="prose prose-lg max-w-none prose-headings:text-foreground prose-p:text-foreground prose-p:leading-relaxed prose-strong:text-foreground prose-code:text-foreground prose-li:text-foreground"
+      dangerouslySetInnerHTML={{ __html: htmlContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)